packages/astro/src/cli.ts

```diff
@@ -76,6 +78,7 @@ function printHelp() {
   --config <path>       Specify the path to the Astro config file.
   --project-root <path> Specify the path to the project root folder.
   --no-sitemap          Disable sitemap generation (build only).
+  --no-drafts           Enable generation of draft pages (build only).
```

Comment:
I'm a little confused by the naming here, this *will* generate draft pages? If so I find `--no-drafts` as a name confusing as that reads to me "do not build drafts".

Reply:
Is there a reason why to add a CLI flag for this at all? Is there a scenario where you want to override what is in your astro config?

Reply:
Regarding naming convention I agree. I can add `!` to invert the CLI flag to match the correct config flag. The config flag + CLI was mentioned in the feature request. I think it's the same topic as with the sitemap.

The idea for the CLI flag is that you might have cases where you want to generate the drafts.
For me the cases could be to build a complete site with draft posts for a staging environment without needing to change the `astro.config.mjs`

Reply:
this is probably most commonly used in 11ty/Jeckll/etc. via CLI flag, but I agree with Matthew that this section is really for special config and this would be better in our normal config file.